src/System.Private.CoreLib/src/Internal/Runtime/Augments/EnvironmentAugments.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Internal.Runtime.Augments
+{
+    /// <summary>For internal use only.  Exposes runtime functionality to the Environments implementation in corefx.</summary>
+    public static class EnvironmentAugments
+    {
+        public static int CurrentManagedThreadId => Environment.CurrentManagedThreadId;
+        public static void Exit(int exitCode) => Environment.Exit(exitCode);
+        public static int ExitCode { get { return 0; } set { throw new PlatformNotSupportedException(); } }
+        public static void FailFast(string message, Exception error) => Environment.FailFast(message, error);
+        public static string[] GetCommandLineArgs() => Environment.GetCommandLineArgs();
```

Comment:
This will cause build break when integrated to TFS.  `Environment.GetCommandLineArgs` was not implemented for .NET Native for UWP yet - it is under CORERT ifdef. We should remove the `CORERT` ifdef in `Environment.cs`. It won't make it work, but it will be good enough to fix the build break.